### ISSUE TITLE
[Feature] Support Mux.Cool

### ISSUE THREAD

**Issue by APT-ZERO** [MetaCubeX/mihomo#1108]
### Verify steps

- [X]  我已经在 [Issue Tracker](……/) 中找过我要提出的请求 I have searched on the [issue tracker](……/) for a related feature request. 
- [X]  我已经仔细看过 [Documentation](https://wiki.metacubex.one/) 并无法找到这个功能 I have read the [documentation](https://wiki.metacubex.one/) and was unable to solve the issue. 

### Description

Mux.Cool is built-in mux of Xray-core
Mux.Cool and Fragment are good unique features of Xray-core that is good if Mihomo add them too

**Comment by lux5am** [comment]
Yeah most server uses xray while most user uses mihomo or v2rayng. Mihomo is good for its easier configuration.

**Comment by Mars160** [comment]
Mux.Cool Plz!

**Comment by hadifarnoud** [comment]
Fragment cripples IRFW. It works great for Iran and Russia!
